src/main/java/com/lorepo/icplayer/client/module/text/TextPrintable.java

```diff
@@ -129,7 +163,20 @@ private String makePrintableInput(String parsedText, boolean showAnswers) {
 
 			span.setInnerHTML(value);	
 			String newValue = "&nbsp;" + span.getString();
-			
+
+			if (this.userAnswers != null && showAnswers) {
+				do {
+					if (userAnswer.equals(answers.next())) {
+						newValue += " ✔ ";

```

Comment:
Mam ambiwalentny stosunek do takich znaczkow - nie wiem czy to nie bedzie mialo potem problemow w dalszych etapach przetwarzania tych plikow - najlepiej byloby umiescic jednak tekst, ktory zostanie zrzucony do utf-8

Reply:
Nie jestem pewien czy do końca rozumiem. Chodzi o to, żeby przekonwertować znaczek ✔ na kod znaku char, zamiast wstawiać go bezpośrednio do stringa czy może skorzystać z pliku .css? Bo z tego co się orientuję, to ten znak jest integralną częścią kodowania utf-8, więc **teoretycznie** nie powinien stanowić problemu.